allotment.go

```diff
@@ -21,6 +21,10 @@ func (a Allotments) Size() int {
 	return len(a) * (AccountIDLength + serializer.UInt64ByteSize)
 }
 
+func (a Allotments) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return workScoreStructure.FactorAllotment.Multiply(a.Size())
```

Comment:
IMO we should take network traffic into account, so we should add a `workScoreStructure.FactorData.Multiply(a.Size()) + ` everywhere.

```suggestion
	return workScoreStructure.FactorData.Multiply(a.Size()) + workScoreStructure.FactorAllotment.Multiply(a.Size())
```

Reply:
Agreed, done.